docs/external-db.md

```diff
@@ -0,0 +1,159 @@
+# External DB integration
+
+Backstage hosts the data in the [PostgreSQL database](https://backstage.io/docs/getting-started/config/database/).
+By default, Operator creates and manages local instance of PostgreSQL in the same as Backstage deployment namespace but it also allows to switch this off and configure external instance instead.
```

Comment:
This talks about the operator but do we need to document this for the helm chart too?  I imagine most of it will be similar

Reply:
@kim-tsao Well, I have instruction for Helm Chart too, just thinking how to organize it not duplicating Postgre SQL types part (same for both). Any suggestions?